client/src/components/chat/chat.tsx

```diff
@@ -90,8 +97,13 @@ export function HistoryChat(args: ScrollingQuestionsParams): JSX.Element {
     }, {});
   });
 
-  const chatData = useSelector<State, ChatData>((s) => s.chat);
+  useEffect(() => {
+    chatData.messages.length === 0
```

Comment:
this seems a little weird...

Do we actually have to trigger a question send at all?

*SOMETHING* must be making a server call that retrieves the mentors' intro responses already, no? Wherever in redux we are capturing the mentors' intro responses and putting them somewhere in redux, can we just also add them to the chat history

Reply:
why did you marked this resolved? Always let the reviewer mark an issue resolved, unless the thing they asked for is extremely straight forward and you actually did it.

The thing I'd prefer to get rid of here is sending a fake question to the server. Do we really need to do that?

Reply:
My bad, I just removed it from the reducer but not sending a fake question. I already removed that part.